WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentService.swift

```diff
@@ -0,0 +1,91 @@
+import Combine
+import Foundation
+import struct Yosemite.Order
+import struct Yosemite.CardPresentPaymentsConfiguration
+
+import UIKit // TODO: remove after update to `ViewControllerPresenting` when #12864 is done
+
+final class CardPresentPaymentService: CardPresentPaymentFacade {
+    let paymentEventPublisher: AnyPublisher<CardPresentPaymentEvent, Never>
+
+    var connectedReaderPublisher: AnyPublisher<CardPresentPaymentCardReader?, Never> {
+        connectedReaderSubject.eraseToAnyPublisher()
+    }
+
+    // I don't actually think we need this, but just in case the service needs to send its own events
+    private let paymentEventSubject = PassthroughSubject<CardPresentPaymentEvent, Never>()
+    private let connectedReaderSubject = CurrentValueSubject<CardPresentPaymentCardReader?, Never>(nil)
+
+    private let onboardingAdaptor: CardPresentPaymentsOnboardingPresenterAdaptor
+
+    private let paymentAlertsPresenterAdaptor: CardPresentPaymentAlertsPresenting
+    private let connectionControllerManager: CardPresentPaymentsConnectionControllerManager
+
+    private let siteID: Int64
+
+    private var cardPresentPaymentsConfiguration: CardPresentPaymentsConfiguration {
+        CardPresentConfigurationLoader().configuration
```

Comment:
Soo... it shouldn't ever change except on a store switch, because it's tied to the store. I think we're keeping the behaviour that the POS session gets recreated on a store switch. 

But yeah, if we wanted to, we could just pass it in to the POS experience when we start and then stick with that, same as siteID. It would probably be better to do that.